src/Razor/perf/Microsoft.AspNetCore.Razor.Performance/Serialization/RazorDiagnosticJsonConverter.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using Microsoft.AspNetCore.Razor.Language;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor.Serialization
+{
+    internal class RazorDiagnosticJsonConverter : JsonConverter
```

Comment:
You shouldn't need to replicate this. We have a converter in the CodeAnalysis.Workspaces dll already.

Reply:
Removed, and updated the Performance.csproj with the file compilation references as we do elsewhere